styling-and-appearance/themes-suite/fluent.md

```diff
@@ -389,42 +385,52 @@ It is possible to disable the ripple effect of the FluentControl through a style
+*  `ScrollViewerScrollBarsMode.Compact`&mdash;The ScrollBars appear always as a narrow sliver.
+*  `ScrollViewerScrollBarsMode.Normal`&mdash;The ScrollBars appear always with their normal size.
 
-*  **ScrollViewerScrollBarsMode.Normal**:  The ScrollBars appear always with their normal size.
+>important The `ScrollBarMode` property and the `ScrollViewerScrollBarsMode` enumeration of the FluentPalette, introduced with the  __2019.1.121__ version of our dlls, are __obsolete__ and __replaced__ by the `ScrollBarsMode` property of the palette and the `ScrollViewerScrollBarsMode` enum under the `Telerik.Windows.Controls.Theming` namespace.
 
->important The __ScrollBarMode__ property and the __ScrollViewerScrollBarsMode__ enumeration of the FluentPalette, introduced with the  __2019.1.121__ version of our dlls, are __obsolete__ and __replaced__ by the __ScrollBarsMode__ property of the palette and the __ScrollViewerScrollBarsMode__ enum under the __Telerik.Windows.Controls.Theming__ namespace.
+__ScrollBar behavior in Auto, Compact and Normal ScrollBarsMode__
 
-#### **Figure 7: ScrollBar behavior in Auto, Compact and Normal ScrollBarsMode**  
 ![ScrollBar in Auto, Compact and Normal ScrollBarsMode](images/fluent-light-scrollbarmode.gif)
 
-__Example 10__ demonstrates how the __ScrollBarsMode__ property of the palette can be set.
-
-#### **[C#] Example 10: Setting the __ScrollBarsMode__ property of the palette to Compact**
+#### __[C#] Setting the __ScrollBarsMode__ property of the palette to Compact__
 {{region styling-appearance-fluent-theme-12}}	
 	FluentPalette.Palette.ScrollBarsMode = Telerik.Windows.Controls.Theming.ScrollViewerScrollBarsMode.Compact;
 {{endregion}}
 
 ## Changing Opacity 
 
-If you need to change the opacity of the disabled and read-only elements, you can now easily do so by using the __DisabledOpacity__ and __ReadOnlyOpacity__ properties of the FluentPalette. The default values are 0.3 and 0.5 respectively. 
+If you need to change the opacity of the disabled and read-only elements, you can now easily do so by using the `DisabledOpacity` and __ReadOnlyOpacity__ properties of the FluentPalette. The default values are __0.3__ and __0.5__ respectively. 
 
-#### __[C#] Example 11: Changing the opacity__		
+#### __[C#] Changing the opacity__		
 {{region cs-styling-appearance-fluent-theme-13}}
 	FluentPalette.Palette.DisabledOpacity = 0.5;
 	FluentPalette.Palette.ReadOnlyOpacity = 0.4;
 {{endregion}}
 	
-#### __[VB.NET] Example 11: Changing the opacity__		
+#### __[VB.NET] Changing the opacity__		
 {{region vb-styling-appearance-material-theme-14}}
 	FluentPalette.Palette.DisabledOpacity = 0.5
 	FluentPalette.Palette.ReadOnlyOpacity = 0.4
 {{endregion}}
 
+>important When modifying fonts, colors, or other resources from the `FluentPalette` and `StyleManager` is used as theming mechanism, the theme's `ResourceDictionary` needs to be merged in __App.xaml__ file to apply the changes.
```

Comment:
This information should be grouped into a new section of the article. Currently, it looks like part of the Changing Opacity  section. Separate this with a new section - for example, "Modifying Palette Resources When Using StyleManager".

This is valid for all of the edited articles in the series.

In this case you can remove the >important and leave the information as a plain text.

Also, creating a section will make it easier to navigate and more visible because you will see the section at the right aside menu in the page.